pkg/util/cluster/cluster.go

```diff
@@ -96,25 +96,26 @@ func New(log *logrus.Entry, environment env.Core, ci bool) (*Cluster, error) {
 		return nil, err
 	}
 
-	scopes := []string{environment.Environment().GraphEndpoint + "/.default"}
-	graphAuthorizer := azidext.NewTokenCredentialAdapter(tokenCredential, scopes)
+	spGraphClient, err := environment.Environment().NewGraphServiceClient(tokenCredential)
```

Comment:
side note: can we rename tokenCredential to have where the token came from as well?  

i.e. fpTokenCredential, msiTokenCredential, etc.